pubsub/rabbitmq/metadata.go

```diff
@@ -76,10 +87,19 @@ func createMetadata(pubSubMetadata pubsub.Metadata) (*metadata, error) {
 		publisherConfirm: false,
 	}
 
-	if val, found := pubSubMetadata.Properties[metadataHostKey]; found && val != "" {
-		result.host = val
-	} else {
-		return &result, fmt.Errorf("%s missing RabbitMQ host", errorMessagePrefix)
+	if val, found := pubSubMetadata.Properties[metadataConnectionStringKey]; found && val != "" {
+		result.connectionString = val
+	} else if val, found := pubSubMetadata.Properties[metadataHostKey]; found && val != "" {
+		result.connectionString = val
+		log.Warn("[DEPRECATION NOTICE] The 'host' argument is deprecated. Use 'connectionString' or individual connection arguments instead.")
```

Comment:
Can you add a link to the Dapr docs too in the log messages, which we'll update? https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-rabbitmq/